metrics/default_metrics/tarantool/init.lua

```diff
@@ -11,6 +11,7 @@ local default_metrics = {
     require('metrics.default_metrics.tarantool.memory'),
     require('metrics.default_metrics.tarantool.spaces'),
     require('metrics.default_metrics.tarantool.fibers'),
+    require('metrics.default_metrics.tarantool.cpu'),
```

Comment:
CPU metrics should not be enabled by default. There are dozens of tools to collect process CPU metrics e.g. telegraf procstat module, prometheus node_exporter etc. to consider before using built in cpu metrics.